cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -45,48 +47,57 @@ private StoreDefinition() {
    * @param tableAdmin the table admin to create the table
    */
   public static void createAllTables(StructuredTableAdmin tableAdmin) throws IOException {
-    ArtifactStore.create(tableAdmin);
-    OwnerStore.create(tableAdmin);
-    NamespaceStore.create(tableAdmin);
-    SecretStore.create(tableAdmin);
-    WorkflowStore.create(tableAdmin);
-    ConfigStore.create(tableAdmin);
-    PreferencesStore.create(tableAdmin);
-    ProvisionerStore.create(tableAdmin);
-    AppMetadataStore.create(tableAdmin);
-    ProfileStore.create(tableAdmin);
-    ProgramScheduleStore.create(tableAdmin);
-    DatasetInstanceStore.create(tableAdmin);
-    DatasetTypeStore.create(tableAdmin);
-    LineageStore.create(tableAdmin);
-    JobQueueStore.create(tableAdmin);
-    TimeScheduleStore.create(tableAdmin);
-    RemoteRuntimeStore.create(tableAdmin);
-    ProgramHeartbeatStore.create(tableAdmin);
-    LogCheckpointStore.create(tableAdmin);
-    UsageStore.create(tableAdmin);
-    FieldLineageStore.create(tableAdmin);
-    LogFileMetaStore.create(tableAdmin);
-    CapabilitiesStore.create(tableAdmin);
-    TetheringStore.create(tableAdmin);
-    AppStateStore.create(tableAdmin);
-    CredentialProviderStore.create(tableAdmin);
-    OperationRunsStore.create(tableAdmin);
+    LOG.debug("system tables creation started.");
```

Comment:
This approach will work logically but from it doesn't look correct from a code readability / pattern. 

Here we are saying "add a DDL statement" , but internally in Postgres, it creating Tables while this command is being run. 

But In Spanner, it will be done later at a moment. 

Overall the code misleads the implementation of Postgres / no sql.

Reply:
Updated the approach, please check.